packages/ui/src/domainModules/collectionMammals/components/RecordNavigationBar/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Button, Icon, Grid } from 'semantic-ui-react'
+
+export const RecordNavigationBar = ({
+  recordNumber,
+  totalRecords,
+  onPreviousRecord,
+  onNextRecord,
```

Comment:
maybe good to add Select to those names so e.g  `onSelectPreviousRecord`, or what do you think?